src/main/java/net/minecraftforge/datafix/ForgeDataFixerCustomFixersModStateTransition.java

```diff
@@ -0,0 +1,220 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2022.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.datafix;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import com.mojang.datafixers.DataFix;
+import com.mojang.datafixers.schemas.Schema;
+import com.mojang.logging.LogUtils;
+import net.minecraft.Util;
+import net.minecraft.util.datafix.DataFixers;
+import net.minecraftforge.common.ForgeSerialModStateTransition;
+import net.minecraftforge.event.datafix.RegisterFixesEvent;
+import net.minecraftforge.fml.IModStateTransition;
+import net.minecraftforge.fml.ModContainer;
+import org.slf4j.Logger;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executor;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+/**
+ * Event handler utility class for managing the events related to custom DFU fixers.
+ * This particular class is used to handle firing of the parallel mod loading events
+ * for the DFU Fixer setup.
+ *
+ * These events are fired immediately after mod construction and before the rest of the game
+ * loads, even before registries are fired.
+ *
+ * THIS CLASS IS FOR INTERNAL USE ONLY!
+ */
+public class ForgeDataFixerCustomFixersModStateTransition
+{
+    private static final Logger LOGGER = LogUtils.getLogger();
+
+    /**
+     * Builds a new mod state transition which can be used to fire the appropriate fixer registration events on the mod bus.
+     *
+     * @return The mod state transition used to add DFU fixers.
+     */
+    public static IModStateTransition get() {
```

Comment:
Formatting: braces on newlines.
```suggestion
    public static IModStateTransition get()
    {
```